packages/compass-connections-navigation/src/styled-navigation-item.tsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import {
+  useConnectionColor,
+  DefaultColorCode,
+} from '@mongodb-js/connection-form';
+
+export default function StyledNavigationItem({
+  colorCode,
+  isSingleConnection,
+  children,
+}: {
+  colorCode?: string;
+  isSingleConnection: boolean;
+  children: React.ReactChild;
+}): React.ReactElement {
+  const { connectionColorToHex, connectionColorToHexActive } =
+    useConnectionColor();
+  const style: React.CSSProperties & {
```

Comment:
should this be a memo? sorry, I know I wrote this bit, but now that I'm having a second look on it.. 😁

Reply:
haha no worries, let me take a look at it :D